### DISCUSSION TITLE
Question: Remote App

### DISCUSSION THREAD

**Discussion by ghost** [neutrinolabs/xrdp#2722] [Q&A] [answered]
Is it actually possible to launch apps remotely through xrdp? i did not find anything about this on your wiki, there seems to be 1 issue/ticket from 2018 where you said it is still in developtement, so im asking if you guys have any updates, this should be for instance a .rdp config for example:

allow desktop composition:i:1
allow font smoothing:i:1
alternate shell:s:rdpinit.exe
devicestoredirect:s:*
disableremoteappcapscheck:i:1
drivestoredirect:s:*
full address:s:foo.domain.bar
prompt for credentials on client:i:1
promptcredentialonce:i:0
redirectcomports:i:1
redirectdrives:i:1
remoteapplicationmode:i:1
remoteapplicationname:s:foobar
remoteapplicationprogram:s:||Programm or PATH to program??

is making of something like this is possible? that you actually just click on it and instead of having a full desktop, you see the login prompt and then just the app, if not, perhaps i could make a different user and make the users start script just that window?

**Comment by ghost** [comment]
Sorry for opening an issue and not a discussion lol, my apologies

**Reply by metalefty** [reply]
No problem. I'm preparing issue template to guide newbies to post questions as discussions.

**Reply by ghost** [reply]
> No problem. I'm preparing issue template to guide newbies to post questions as discussions.

good thanks, any update on functionalities of remote app on xrdp? i saw a tread that it was being worked on (i think 2018), sadly i didnt find anything else on here or wiki, is it even possible? is it going to be an option in the future?

**Reply by matt335672** [reply, marked as answer]
There's no update yet I'm afraid.

We're working on other things which are higher priority, and at the moment we've got no spare effort to look into this.

**Reply by ghost** [reply]
> There's no update yet I'm afraid.
> 
> We're working on other things which are higher priority, and at the moment we've got no spare effort to look into this.

thank you for the reply, ive been looking effortlessly for a similar functionality, there seems to be nothing like this outthere sadly...
